routers/web/repo/compare.go

```diff
@@ -874,7 +874,7 @@ func ExcerptBlob(ctx *context.Context) {
 		}
 	}
 	ctx.Data["section"] = section
-	ctx.Data["fileName"] = filePath
+	ctx.Data["NameHash"] = base.EncodeSha1(filePath)
```

Comment:
It might be `FileNameHash` because this field doesn't belong to any struct.

`fileName` => `FileNameHash` seems more intuitive